src/colorizer/ColorizeCanvas.ts

```diff
@@ -329,7 +329,7 @@ export default class ColorizeCanvas {
     }
     this.dataset = dataset;
     if (this.dataset.outliers) {
-      this.setUniform("outlierData", this.dataset.outliers);
+      this.setUniform("outlierData", packDataTexture(Array.from(this.dataset.outliers), FeatureDataType.U8));
```

Comment:
Going back over `packDataTexture` and realizing that it probably ought to be accepting `TypedArray`s rather than regular JS arrays... not on you to fix for this PR, just an observation